OpenRA.Mods.Common/Traits/Player/TechTree.cs

```diff
@@ -94,14 +94,13 @@ public bool HasPrerequisites(IEnumerable<string> prerequisites)
 			}
 
 			// Add buildables that have a build limit set and are not already in the list
```

Comment:
This comment is wrong now as you removed
```csharp
!ret.ContainsKey(a.Actor.Info.Name)
```

Reply:
Thanks for point out, I will correct it.